custom-login/README.md

```diff
@@ -52,13 +52,20 @@ Now start the app server:
 ng serve
 ```
 
+> **Note:** If you're on a windows machine, you might get an error similar to
+> ```
+> ng serve contains error Unknown browser query `basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")`
```

Comment:
This still renders a bit funny with the overflow text. You can see that [here](https://github.com/okta/samples-js-angular/blob/0e4d40bf6455dddd8d392ccd2bcf5ba7316c2dab/custom-login/README.md#running-this-example): 

Suggestion:

----
>  **Note:** If you're on a windows machine, you might get an error similar to
> `ng serve contains error Unknown browser query basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")`
>
> It's a [known issue](https://github.com/angular/angular-cli/issues/5075) on windows due to one of the dependencies we use. To resolve the issue, search for browserslist and browserslist.cmd files in your node_modules. Delete both the files and start the app server again.